### PR TITLE
feat: add issue template request promotion

### PR DESCRIPTION
**Pull request by LuisRickert**
---
name: Add request KIT Promotion Issue Template
title: feat add issue templates to request promotion
---

## Description

With this PR we introduce the issue templates to request a promotion for an existing KIT. Raising such an issue will trigger the review process.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue-request-promotion-incubating.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        - label: Whitepaper
+    validations:
+      required: true
+  - type: textarea
+    id: comments
+    attributes:
+      label: comments
+      description: "Leave a comment or additional information!"
+
+
```
Comment (FaGru3n) [inline .github/ISSUE_TEMPLATE/issue-request-promotion-incubating.yaml:41]: i think just a single blank line will be ok 😉

File: .github/ISSUE_TEMPLATE/issue-request-promotion-graduated.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+name: KIT-Promotion-Graduated
+description: Request to promote your KIT
+title: "[Promote KIT To Graduated]: "
+labels: ["kit-promotion"]
+assignees:
+  - octocat
```
Comment (FaGru3n) [inline .github/ISSUE_TEMPLATE/issue-request-promotion-graduated.yaml:6]: I would leave this out if it is not clear who will be responsible @octocat seems for me to generic

Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/issue-request-promotion-graduated.yaml]: hi @LuisRickert also your could reference it and make it as resolved 😉 
[chore: remove octocat as asignee, remove unneccessary information](https://github.com/eclipse-tractusx/sig-project-managment/pull/6/commits/e2ad74d20343f14dabadc8ce903cc8aaa0b6f805)

File: .github/ISSUE_TEMPLATE/issue-request-promotion-graduated.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      description: Link to your KIT-Repo
+    validations:
+      required: true
+  - type: checkboxes
+    id: artefacts_and_Deliverables
+    attributes:
+      label: "Artefacts & Deliverables"
+      description: "Tick the provided artefacts and deliverables"
+      options:
+        - label: Standarts
```
Comment (carslen) [inline .github/ISSUE_TEMPLATE/issue-request-promotion-graduated.yaml:26]: This is not art but Standard ;)

**Review by carslen — CHANGES_REQUESTED** [review]

**Comment by FaGru3n** [comment]
Hi @LuisRickert because this `sig-release` repo is getting more usage of planning and overviewing the release path and processes do you think a new repo would be nice, like "sig-kit" in my opnion it makes it easier to keep on track with the release process and planning things ... and also for the requests against it...?

beside the fact more and more KIT´s will be developed... so after the time of requests against

**Comment by LuisRickert** [comment]
> Hi @LuisRickert because this `sig-release` repo is getting more in usage of planning and overviewing the release path and processes do you think a new repo would be nice, like "sig-kit" in my opinion it makes it easier to keep on track with the release process and planning things ... and also for the requests against it...?
> 
> beside the fact more and more KIT´s will be developed... i would prefer a special repo for a special interest group 😉

Well i guess then a dedicated Repository is the right was

**Comment by RolaH1t** [comment]
@FaGru3n after several discussions (e.g. with @danielmiehle) I can confirm, that we will "park" the idea of an additional/separate RePo for KITs for the moment. Please work with @kelaja to process all release-relevant issues for KITs also within the SIG Release RePo. We will review the idea towards the and of PI10, once initial learnings can be applied.

**Comment by RolaH1t** [comment]
@LuisRickert please dismiss this PullRequest. @danielmiehle will also confirm here, that the initial idea of a template to promote a KIT shall not be used for the time being. Thx, Roland

**Comment by FaGru3n** [comment]
I will close the PR according to the message from @RolaH1t

**Pull request closed without merging**